CHANGELOG.md

```diff
@@ -119,6 +119,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     * Updated precision attributes in `DeepSpeedPlugin` ([#10164](https://github.com/PyTorchLightning/pytorch-lightning/pull/10164))
     * Added the ability to return a result from rank 0 in `DDPSpawnPlugin.spawn` ([#10162](https://github.com/PyTorchLightning/pytorch-lightning/pull/10162))
     * Added `pytorch_lightning.lite` package ([#10175](https://github.com/PyTorchLightning/pytorch-lightning/pull/10175))
+- Added `ByteCounter` to calculate model_size ([#10123](https://github.com/PyTorchLightning/pytorch-lightning/pull/10123))
```

Comment:
I suggest to mention this in the changed section, that we are now using a byte counter instead of file io.